verification.py

```diff
@@ -1651,6 +1649,7 @@ def _run(self, xi_model, **kwargs):
             model_str = "hudson"
             if model != "hudson":
                 model_str = "Xi"
+                simulate_args["Ne"] = 2*(math.sqrt(int(simulate_args["Ne"])))
```

Comment:
A comment here would be helpful I think. "We need to rescale Ne by blah because blah blah".

Reply:
Okay. I hope the comment is clear.

Reply:
Perfect. Just need to get the formatting fixed now and we're good. All you really need to do is run "black" on the code, like
```
python3 -m pip install black
python3 -m black verification.py
git commit -a --amend
```
and that should do it.

Reply:
Thanks ! Sorry  for all the trouble, as  I thought is was a small commit I went through the GitHub UI.